app/res/values-xlarge/dimens.xml

```diff
@@ -12,4 +12,5 @@
     <dimen name="row_padding_vertical">8sp</dimen>
     <dimen name="row_padding_horizontal">12sp</dimen>
 
+    <dimen name="edit_text_question_widget_compact_width">120dp</dimen>

```

Comment:
If this needs to be overridden in `values-xlarge` and `values-large`, should it be overridden for other sizes as well?

Reply:
Yeah. I think `values-sw720dp-land` will correspond to `xlarge` bucket as well. So going to add it there. Apart from that it looks good to me.

Reply:
Actually no, since android will automatically use the best matching dimension value depending on the screen size so we don't need  to specify it in `values-sw720dp-land` as well